authzed/api/v1/core.proto

```diff
@@ -76,6 +76,15 @@ message ZedToken {
   } ];
 }
 
+// Cursor is used to provide resumption of listing between calls to APIs
+// such as LookupResources.
+message Cursor {
```

Comment:
Do we need a message for this or should we just use a string field in the parent message?

Reply:
I like having a message in case we need to make semantics changes to it, like we do for ZedToken

Reply:
Yeah, that's the idea